### DISCUSSION TITLE
ray tracing and dlss for rtx

### DISCUSSION THREAD

**Discussion by cathaysia** [Heroic-Games-Launcher/HeroicGamesLauncher#1193] [General]
ray tracing is disabled be default and games likes Control or BattleField V cannot enable it. But I find some codes to enable it:

```none
VKD3D_CONFIG=dxr PROTON_HIDE_NVIDIA_GPU=0 PROTON_ENABLE_NVAPI=1
```

that work for me. (Both Control or BattleField V). Can someone add it to launcher?

ps: the code source is https://www.protondb.com/app/1238810

**Comment by cathaysia** [comment]
emm, it can unlock dlss also.

**Comment by flavioislima** [comment]
We have a feature request for that already. The thing is that we need a solution that will work for both Proton and Wine. I believe this one works for proton only.
For Wine we will need to implement NVAPI.
